Cargo.toml

```diff
@@ -12,7 +12,7 @@ cortex-m = "0.7"
 cortex-m-rt = "0.6"
 panic-reset = "0.1"
 nb = "1"
-panel-protocol = { git = "https://github.com/tonarino/panel-protocol.git", rev = "0.4" }
+panel-protocol = { git = "https://github.com/tonarino/panel-protocol.git", branch = "set-pulse-interval" }
```

Comment:
This can be reverted once https://github.com/tonarino/panel-protocol/pull/8 is merged.